src/cascadia/TerminalSettingsModel/Profile.idl

```diff
@@ -80,27 +68,15 @@ namespace Microsoft.Terminal.Settings.Model
         INHERITABLE_PROFILE_SETTING(String, StartingDirectory);
         String EvaluatedStartingDirectory { get; };
 
-        INHERITABLE_PROFILE_SETTING(String, BackgroundImagePath);
-        String ExpandedBackgroundImagePath { get; };
+        IAppearanceConfig DefaultAppearance { get; };
+        INHERITABLE_PROFILE_SETTING(IAppearanceConfig, UnfocusedAppearance);

```

Comment:
* * * W A R N I N G

Did we want to have an UnfocusedAppearance node at every level? Remember: this makes it so that if you set an unfocused appearance on `defaults`, it wins over the focused appearance of the profile.

Reply:
Fall back on what the agreed spec says

Reply:
Yes we do! We will inherit the UnfocusedAppearance as a single setting, which means
- this profile defines an unfocused appearance: just use that
- this profiles does not define an unfocused appearance, use unfocused appearance from `defaults` if it exists